plenum/cli/cli.py

```diff
@@ -255,13 +255,12 @@ def __init__(self, looper, basedirpath, nodeReg=None, cliNodeReg=None,
         else:
             msg = """
             No information is found which can be used to connect to the nodes.
-             This indicates an error. Check if the file containing genesis
-             transactions (which has name specified in config as
-             `poolTransactionsFile`) is present in your base directory
-             which can be found in the config as `baseDir`, if not then get
-             this file from the github repository under `/data` and paste it
-             in location `baseDir`. The github url repository is at {}
-            """.format(self.githubUrl)
+            This indicates an error. Check if the file containing genesis
+            transactions (the `poolTransactionFile` in the configuration file)
+            is present in your base directory which can be found in the config
+            as `baseDir`, if not then get this file from the github repository
+            under `/data` and paste it in location `baseDir`. The github url is
+            {}.\n""".format(self.githubUrl)
```

Comment:
I am seeing we are telling user to do few things instead of directly giving path and github urls for required things. Also the message came in between lines where it is not that obvious, so few formatting needs to be done. I can help you design that. Its 10 min of work.